05-Photometry/01-aperture_basics.ipynb

```diff
@@ -801,7 +801,7 @@
     "sky_coord = wcs.pixel_to_world(x, y)  # a SkyCoord object\n",
     "\n",
     "# we can add the astropy SkyCoord object directly to the table\n",
-    "phot['sky_coord'] = sky_coord\n",
+    "phot['sky_center'] = sky_coord\n",
```

Comment:
This is to be consistent with photutils column name when it does return sky coordinates.